parser/src/lib.rs

```diff
@@ -125,6 +125,30 @@ pub fn unescape_string(s: &str) -> String {
     result
 }
 
+use powdr_ast::parsed::{PILFile, PilStatement};
+fn pil_statement_clear_source_ref(stmt: &mut PilStatement) {
```

Comment:
Why did you move this out of the test module?

Reply:
Ah I see. Can you create a new file `test_utils.rs` or something and move it there?